routes/location.js

```diff
@@ -395,23 +396,27 @@ function getK8sCloudMetadata(callback) {
             console.log(`RESULT: ${metaData}`);
             console.log('No more data in response.');
 
-	    if (metaData.spec.providerID) {
-            	var provider = metaData.spec.providerID;
-            	cloudName = String(provider.split(":", 1)); // Split on providerID if request was successful
-	    }
-
+	        if (metaData.spec.providerID) {
```

Comment:
nit: is this a spacing change only? I'm not sure it lines up with the other logic.

Reply:
It seems so, it was showing okay in my editor, it's fixed now.